DI all the things, everything is great except it doesn't actually work

THEjoezack opened the pull request.

THEjoezack committed:
DI all the things, everything is great except it doesn't actually work

THEjoezack committed:
Cleaning up some dependencies

Kritner commented:
And just to be clear, it's only the console logging that doesn't work right? :P
DI works, and debug logging, for whatever reason works.  You can even see a "Console" and "Debug" logged attached when stepping into the class.

/shrug

Kritner committed:
@jz would you believe me if I told you console logging *was* working?
- logging is apparently handled on a separate thread and without the console.readkey the app terminates too quickly for the console logging to process.
- as per https://github.com/aspnet/Logging/issues/631

Kritner committed:
Adds "None" case to enum.

THEjoezack committed:
Merge pull request #3 from Kritner/DI

@jz would you believe me if I told you console logging *was* working?
